src/redux/combineReducers.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux';
+
```

Comment:
Kindly install `redux` as well as `npm install redux`. If you look at your `import` for `combineReducers` and `createStore `, they are from `redux` and if `redux` is not installed you will run into errors in the future for a missing package.

NOTE: `redux` and `react-redux` are two separate packages. I know that you've installed `react-redux`, but you'll also need to install `redux` too to get your imports to work.